benchmark/world_benchmark.dart

```diff
@@ -4,15 +4,15 @@ import 'package:benchmark/benchmark.dart';
 void main() {
   group('World', () {
     group('Without world creation', () {
-      World world;
+      World? world;
 
       setUpEach(() => world = World());
 
       tearDownEach(() => world = null);
 
       benchmark('World with 100000 entities', () {
```

Comment:
Completely unrelated to the PR, but we should really start doing this in Forge2D instead of the crude benchmarking code that is in there now.

Reply:
I personally find my implementation both in code and output more readable.

![image](https://user-images.githubusercontent.com/15887627/110208563-35e84a80-7e88-11eb-96d1-8ff7e3502c1b.png)

Reply:
Indeed, much better!